exporters/otlp/src/otlp_exporter.cc

```diff
@@ -30,13 +31,28 @@ void PopulateRequest(const nostd::span<std::unique_ptr<sdk::trace::Recordable>>
   }
 }
 
+static std::string get_file_contents(const char *fpath)
+{
+  std::ifstream finstream(fpath);
+  std::string contents;
+  contents.assign((std::istreambuf_iterator<char>(finstream)), std::istreambuf_iterator<char>());
+  finstream.close();
+  return contents;
+}
+
 /**
  * Create service stub to communicate with the OpenTelemetry Collector.
  */
 std::unique_ptr<proto::collector::trace::v1::TraceService::Stub> MakeServiceStub(
-    std::string endpoint)
+    const OtlpExporterOptions &options)
 {
-  auto channel = grpc::CreateChannel(endpoint, grpc::InsecureChannelCredentials());
+  auto channel = grpc::CreateChannel(options.endpoint, grpc::InsecureChannelCredentials());
```

Comment:
should we be creating this insecure-channel only when `use_ssl_credentials` is false ?

Reply:
I thought since use_ssl_credentials is false by default, behavior is as original code, please let me know if you still would like it to be in else clause instead.

Reply:
Does it mean that in case of errors like cert file missing or not-readable, we should allow using insecure channel even though the `use_ssl_credentials` is true ?

Reply:
oh I get it now, will address.

Reply:
addressed.